src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -81,7 +81,17 @@ public static void ProcessStatus(IConsole console)
         {
             int GetColumnWidth(IEnumerable<int> fieldWidths)
             {
-                int largeLength = Console.WindowWidth / 2 - 16;
+                int consoleWidth = 0;
+                if (Console.IsOutputRedirected)
+                {
+                    consoleWidth = Int32.MaxValue;
+                }
+                else
+                {
+                    consoleWidth = Console.WindowWidth;
+                }
+                int extra = (int)Math.Ceiling(consoleWidth*0.05);
+                int largeLength = consoleWidth / 2 - 16 - extra;
                 return Math.Min(fieldWidths.Max(), largeLength);
```

Comment:
Only question here is - why the divide console width by 2. There's 4 columns. Granted, the first two should likely not have as much of the console width as the other two, but you can end up with a table wider than the console with this computation.

Reply:
@hoyosjs that is a great point. My thought process was as follows: there are two columns that will be taking up the largest length of the console width. We want as much of those columns as possible. Like you mentioned, the other two columns are negligible. On my machine, the pid can be up to 6-8 ish characters, and then the name is a few more characters as well. That is the motivation behind subtracting 16. Doing that **after** dividing by 2 is like subtracting 32 characters from the entire console length. Then, the motivation behind subtracting extra was looking at the ratio of number of characters longer than the width over the total width of console for various console widths. The ratio was pretty consistent, so I used it.

Reply:
I don't have a number for you, but for example "SqlToolsResourceProviderService" is 31, and 6 for the a pid, and then couple spaces in between columns we are looking at ~40. Not sure how we didn't overflow in that case, but that's a problem to take a look separately. The 16 there looked like a magic constant, adding a const and/or comment would probably be a good idea; I assumed it was to give some whitespace room. That didn't make as much sense as now we are removing 5 chars for every 100 of the console width. I still think we should take this and clean all that afterwards.